frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -226,6 +226,10 @@ export const HorizontalNav = React.memo((props: HorizontalNavProps) => {
 
   const routes = pages.map((p) => {
     const path = `${props.match.url}/${p.path || p.href}`;
+    const componentName = p.component?.name || '';
```

Comment:
@zherman0 can you describe why we need the `p.component.name` check?

Reply:
@spadgett - I needed a way programmatically to tell the difference between the standard "Subscriptions' and the operand's "Subscriptions".  By looking at the objects, I noticed that the component name field is not set by the operand CRD but it is set on the regular menu items.  I am always open to other ways to know which "Subscription" object is which.

Reply:
![fireworks](https://user-images.githubusercontent.com/18728857/81455632-fb6d2e80-914c-11ea-8bcc-871b2cdff34f.gif)

Reply:
Oh, I didn't notice this was in the base horizontal nav module. This logic should *only* be applied to CSVs. We need to move it here:

https://github.com/spadgett/console/blob/a4048cb8d8a910ffc538e55641d12fbdd1784df7/frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx#L971-L984

From that context, you know which tabs are for operands.

Reply:
That is a much better place to make the change, thanks.

Reply:
@spadgett - New code is pushed